prosemirror-lwdita/src/document.ts

```diff
@@ -112,10 +112,9 @@ export const NODES: Record<string, (value: JDita, parent: JDita) => any> = {
  * Transforms the JDita document into a proper ProseMirror document
  *
  * @param value - The JDita node
- * @param parent - The parent JDita node
  * @returns The transformed JDita node
  */
-function defaultTravel(value: JDita, parent: JDita): any {
+export function defaultTravel(value: JDita): any {
```

Comment:
Why does this now need to be `exported`?

Reply:
It does not, this was an old change before we introduced the escape hatch 
```ts
//Escape hatch for Unit Testing due to a lack of “package-private” accessibility scope in TypeScript
export const _test_private_document = {
  deleteUndefined,
  defaultTravel,
  travel,
}
```
Good catch @adamretter